### PR TITLE
(#75) Fixed pager's horizontal ruler on Manual view

### LINKED ISSUE #75: At tablet and mobile breakpoint, double horizontal rule should not be displayed at the top when pager does not show up. (Manual)
### Issue description
See #71 

Make these changes on the release branch release/v1.0.0

> **ESTIMATE** 2

### Steps to reproduce the issue

1.  
2. 
3. 

### What's the expected result?
-

### What's the actual result?
-

### Additional details / screenshot
- Screenshot

### Related Tickets
* Issue #9999
* Issue #9999

### PR DESCRIPTION
**Pull request by KateMashkinaNIH**
- removed hr tag
- updated Manual.css
Closes #75

### CODE REVIEW

**Comment by KateMashkinaNIH** [comment]
@mworrest  @VictoriaSunNIH @alinai here are the urls for review ( remember to check both Desktop and tablet or mobile breakpoints)
if you need to cover jumpMenu to at any point you can append `&hideJumpMenu=true` to the end of an URL

1. Pager present - https://react-app-dev.cancer.gov/clinical-trials-listing-app/pr-76?cfg=4
2. No pager no intro text - https://react-app-dev.cancer.gov/clinical-trials-listing-app/pr-76?cfg=3
3. No pager with intro text - https://react-app-dev.cancer.gov/clinical-trials-listing-app/pr-76

**Comment by alinai** [comment]
Looks good to me on Chrome, FF, IE and Edge. Thanks @KateMashkinaNIH!

**Comment by VictoriaSunNIH** [comment]
Looks good to me too, thanks @kate-mashkina ! Updated label to passed. 

cc: @mworrest

**Comment by mworrest** [comment]
Looks good. I only see the one rule.

File: src/views/Manual/Manual.scss [hunk 1]
```diff
@@ -25,20 +24,29 @@
...
-	display: none;
+.list {
+	border-top: 1px solid #ddd;
+	margin: 1em 0 1em;
 }
 
 @media only screen and (min-width: 1024px) {
-	.list + hr {
+	.list {
 		display: block;
```
Comment (arcepaul) [inline src/views/Manual/Manual.scss:42]: I know you didn't add this but this display:block is unnecessary

File: src/views/Manual/Manual.scss [hunk 2]
```diff
@@ -25,20 +24,29 @@
 
 		&__pager {
 			float: right;
```
Comment (arcepaul) [inline src/views/Manual/Manual.scss:26]: :| Again,  I know you didn't do this but could we convert this to flex?  Everything else is flex goodness.

**Comment by mworrest** [comment]
Looks good to me on these urls 
Pager present - https://react-app-dev.cancer.gov/clinical-trials-listing-app/pr-76?cfg=4
No pager no intro text - https://react-app-dev.cancer.gov/clinical-trials-listing-app/pr-76?cfg=3
No pager with intro text - https://react-app-dev.cancer.gov/clinical-trials-listing-app/pr-76

**Review by arcepaul — APPROVED** [review]

**Review by seyilonge-nci — APPROVED** [review]

**Pull request merged**